### DISCUSSION TITLE
Pijuice Zero max continuous voltage J4 port

### DISCUSSION THREAD

**Discussion by kvanwezemael** [PiSupply/PiJuice#859] [Q&A]
Hello,

I was wondering what the max voltage of the J4 input port is beyond the specified range. 

It is clearly marked that the input voltage should be between 4 and 10V, and normally my client will connect a 7,4V battery to the Pijuice Zero.
However it could occur that a 12 or 14,8V source battery is connected by accident.

Will overvolting to 14,8V damage the circuitry? From the BQ24160RGET charge IC datasheet I get that 20V is rated, but I do not know if other components are connected to this line except for the battery charger circuit who might get damaged.

Many thanks for your help,

Kris

**Comment by tvoverbeek** [comment]
@kvanwezemael IIf you check the schematic (https://github.com/PiSupply/PiJuice/blob/master/Documentation/PiJuice_Schematic.PDF) you see that the J4 input VIN goes straight to the BQ charge ic (via a ferrite bead). For the other components connected to VIN I do not know.

**Reply by kvanwezemael** [reply]
Hello Tvoverbeek!

Thank you for your answer and getting back to me so guickly. 
According to the datasheet the max allowed voltage on the IN pin is 20V, but over voltage protection starts from 10,5V

Best regards,
Kris
